setup-bounce-handler-deploy-file.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+# Sets up the s3 deploy bucket with blue and green files for the bounce handler
+
+# Usage
+#   ./setup-bounce-handler-deploy-file.sh
+
+./check-env-variables.sh \
+  "ENV" \
+  "EFCMS_DOMAIN" \
+  "AWS_ACCOUNT_ID" \
+  "AWS_ACCESS_KEY_ID" \
+  "AWS_SECRET_ACCESS_KEY"
+
+BUCKET_NAME="${EFCMS_DOMAIN}.efcms.${ENV}.us-east-1.lambdas"
+
+aws s3 cp "s3://${BUCKET_NAME}/bounce_handler_blue.js.zip" "s3://${BUCKET_NAME}/bounce_handler_green.js.zip"
```

Comment:
Could the colors be set dynamically using the `get-deploying/current-color.sh` scripts?

For example, in `setup-s3-maintenance-file.sh' 😄

Reply:
y'know i was hoping this thing existed, and i'm so happy to see that it does. ty! added.